src/app/containers/Metadata/index.jsx

```diff
@@ -179,9 +189,9 @@ const MetadataContainer = ({ metadata, promo }) => {
         title={getTitle(promo)}
         twitterCreator={twitterCreator}
         twitterSite={twitterSite}
-        type={assetType}
+        type={deepGet([pageType, 'openGraph'], pageTypeMetadata)}
```

Comment:
Not the keenest, as this could return null for unknown page types, and the component has it as a required prop, but will do for now